packages/documentation/src/content/docs/overview/concepts/accounting.mdx

```diff
@@ -1,139 +1,261 @@
 
 ### Liquidity accounts
 
-A liquidity account holds a non-negative balance, with Rafiki ensuring that total debits do not exceed total credits.
+Liquidity accounts are used to track deposits, withdrawals, and transfers that occur during the course of a transaction. Liquidity accounts are provided for assets, peers, and payments.
 
-There is one liquidity account for each of the following resources:
+Liquidity accounts hold either a zero or a positive balance. Rafiki ensures that the total debits to a liquidity account will not exceed the account's total credits.
 
-- Asset
-- Peer
-- Wallet address
-- Incoming payment
-- Outgoing payment
+<div class="overflow-table" style="border-bottom: 1px solid var(--sl-color-gray-6);">
 
-### Settlement accounts
+| Account type | What the account represents | Number of accounts |
+| ------------------------- | --------------------------- | ------------------ |
+| [Asset liquidity](#asset-liquidity-accounts) | The value, denominated in a given asset, that Rafiki has at its disposal for sending or forwarding ILP packets | One per asset |
+| [Peer liquidity](#peer-liquidity-accounts) | The credit line, denominated in the asset of your peering relationship, that you extend to a peer | One per peer |
+| [Incoming payment liquidity](#incoming-payment-liquidity-accounts) | The value received from a completed incoming payment | One per incoming payment |
+| [Outgoing payment liquidity](#outgoing-payment-liquidity-accounts) | The value that Rafiki will attempt to send in an outgoing payment | One per outgoing payment |
+| [Wallet address liquidity](#wallet-address-liquidity-accounts) | The value of an incoming payment received to a wallet address via either SPSP or Web Monetization | One per wallet address |
+
+</div>
+
+#### Asset liquidity accounts
+
+Asset liquidity ensures there is enough value available to handle a foreign exchange transaction between you and a peer. You have one asset liquidity account for each asset you transact in.
+
+An asset liquidity account represents the value that Rafiki has at its disposal for sending or forwarding ILP packets. The amount in an asset liquidity account increases when packets are received and decreases when packets are sent/forwarded. Any transaction that would result in a negative balance will fail. 
+
+:::note
+If you and your peer transact in the same asset (there's no currency conversion) and you both provide your customers only with wallet addresses denominated in that asset, then there will be no movement info/from the corresponding asset liquidity account. In this scenario, the transaction occurs over Interledger's Simple Payment Setup Protocol (SPSP).
```

Comment:
```suggestion
If you and your peer transact in the same asset (there's no currency conversion) and you both provide your customers only with wallet addresses denominated in that asset, then there will be no movement info/from the corresponding asset liquidity account.
```
Applicable to normal peer-to-peer packet exchange over ILP

Reply:
Removed SPSP line. Also added a bit more to the "For example" portion of the note.